app/domain/conjur/validate_trusted_proxies.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'command_class'
+
+module Conjur
+
+  ValidateTrustedProxies ||= CommandClass.new(
+    dependencies: {
+      fetch_trusted_proxies:  ::Conjur::FetchTrustedProxies.new,
+      logger:                 Rails.logger
+    },
+    inputs:       %i(account proxy_list)
+  ) do
+
+    def call
+      fetch_trusted_proxies_list
+      validate_proxies
+    end
+
+    private
+
+    def fetch_trusted_proxies_list
+      @trusted_proxies_list ||= @fetch_trusted_proxies.(
+        account: @account
+      )
+    end
+
+    def validate_proxies
+      return if @trusted_proxies_list.blank? || @proxy_list.blank?
+
+      @logger.debug(LogMessages::Conjur::ValidatingProxyList.new(@proxy_list.map {|ip| ip.to_s}))
```

Comment:
how will this log look like? can you show me an example?

Reply:
`CONJ00043D Validating proxy list [1.1.1.1, 2.2.2.2]`